src/Repository/RawPaymentMethodRepository.php

```diff
@@ -59,6 +59,11 @@ private function insertConfiguration(string $paymentName, int $paymentMethodId):
         $configValue = ['mode' => 'off'];
 
         switch ($paymentName) {
+            case 'klarna':
+            case 'tinka':
+                $configValue['financial_warning'] = 1;
```

Comment:
save financial_warning as boolean, so below you check if its true or false and not === 1